apps/storybook/src/LineVis.stories.tsx

```diff
@@ -65,15 +65,34 @@ type Story = StoryObj<typeof meta>;
 
 export const Default = {
   render: (args) => {
-    const { dataArray, domain, scaleType, errorsArray, auxiliaries } = args;
+    const {
+      dataArray,
+      domain,
+      scaleType,
+      errorsArray,
+      auxiliaries,
+      showErrors,
+      ignoreValue,
+    } = args;
 
     // If story doesn't provide `domain`, compute it automatically
     const auxArrays = (auxiliaries || []).map(({ array }) => array);
+    const auxErrorsArrays = (auxiliaries || []).map(({ errors }) => errors);
+
     const effectiveDomain =
       domain ||
       getCombinedDomain([
-        getDomain(dataArray, scaleType, errorsArray),
-        ...getDomains(auxArrays, scaleType),
+        getDomain(
+          dataArray,
+          scaleType,
+          showErrors ? errorsArray : undefined,
+          ignoreValue,
+        ),
+        ...getDomains(
+          auxArrays,
+          scaleType,
+          showErrors ? auxErrorsArrays : undefined,
```

Comment:
We don't pass `ignoreValue` here, as the expected behaviour for auxiliary signals is not clear. Something to discuss in the future if the user need arises.